server/src/db.ts

```diff
@@ -59,3 +59,9 @@ export async function init(): Promise<void> {
 export async function getRooms(): Promise<Room[]> {
   return dbAll(`SELECT * FROM rooms`) as Promise<Room[]>;
 }
+
+export async function createRoom(
+  roomRequest: CreateRoomRequest
+): Promise<void> {
+  await dbRun(`INSERT into rooms (name) values ($1)`, [roomRequest.name]);
```

Comment:
Nit: let's keep our capitalization consistent on SQL keywords. I prefer to capitalize SQL keywords myself (and keep variables lower case).